core/versions_2.0.md

```diff
@@ -0,0 +1,71 @@
+# Workflow versioning 2.0
+
+This proposal introduces two key changes to what is called `getVersion` in the existing SDKs, but will likely be renamed in subsequent ones (more on naming later). The key changes are:
+
+* Introduce a different API for versioning that does not involve a version range, but instead simply returns a boolean indicating if a named change has happened or not. Ex: `hasChange("change-id")`. TBD if we would eliminate the range-based API, or leave it for advanced users.
+* It will no longer be valid to replay a workflow history containing a version marker against code which does not have a corresponding call to the version API.
```

Comment:
I wonder if that could make it harder to evolve workflows for a long period of time, as there would be no way to remove any checks for any versions anymore?